### PR TITLE
NFlowJS 

### PR DESCRIPTION
**Pull request by matejgrcic**
run nflowjs method

### CODE REVIEW

File: experiments/fishyscapes.py [hunk 1]
```diff
@@ -248,6 +250,129 @@ def wrapper(image):
...
+
+    _run.info['ood_ratio_rev1'] = fs.evaluate(wrapper, data)
+
+
+@ex.command
+def nflowjs(testing_dataset, _run, _log, validation=False):
+    # BELOW, IMPORT ANY OF YOUR NETWORK CODE
+    from deeplabv3 import DeepWV3PlusTH
+
+    model = DeepWV3Plus(num_classes=19).cuda()
```
Comment (hermannsblum) [inline experiments/fishyscapes.py:316]: I cannot import this model, and if I change this to `DeepWV3PlusTH`, some weights are missing in the model you submitted.

Comment (matejgrcic) [reply experiments/fishyscapes.py:316]: Sorry, fixed it with the latest commit.

**Comment by hermannsblum** [comment]
Thanks for the fix! Another error came up:

```

Traceback (most recent calls WITHOUT Sacred internals):

  File "experiments/fishyscapes.py", line 373, in nflowjs

    _run.info['nflowjs'] = fs.evaluate(wrapper, data)

  File "/usr/local/lib/python3.6/dist-packages/bdlb/fishyscapes/benchmark.py", line 124, in evaluate

    uncertainties.append(estimator(batch['image_left']).numpy())

  File "experiments/fishyscapes.py", line 368, in wrapper

    probs = apply_jsd(logit)

  File "experiments/fishyscapes.py", line 320, in apply_jsd

    out_probs = F.log_softmax(logit / 2., dim=1)

NameError: name 'F' is not defined
```

**Comment by matejgrcic** [comment]
Fixed. Sorry, I didn't manage to set up the evaluation locally. I committed without testing 😢

**Pull request closed without merging**